lms/models/json_settings.py

```diff
@@ -85,6 +84,9 @@ def get(self, group: str, key: str, default=None):  # type: ignore[override]
         """
         return super().get(group, {}).get(key, default)
 
+    def get_setting(self, setting: JSONSetting):
+        return self.get(setting.group, setting.key, setting.default)
```

Comment:
From a Slack conversation I understand that in future all setting access should go through this method, in which case `ApplicationSettings` would no longer need to subclass `JSONSettings` and that super-class could become an implementation detail.